instant-markdown-d

```diff
@@ -59,7 +59,9 @@ function httpHandler(req, res) {
         newHTML += data;
       });
       gfm.on('exit',function(ecode){
-        socket.emit('newContent', newHTML);
+        if (socket) {
+          socket.emit('newContent', newHTML);
+        }
```

Comment:
Yeah, it was not easily reproducible unfortunately. However this saves the server from crashing when it is defined.